src/checkqueue.h

```diff
@@ -147,7 +159,7 @@ class CCheckQueue
     CCheckQueue& operator=(CCheckQueue&&) = delete;
 
     //! Wait until execution finishes, and return whether all evaluations were successful.
```

Comment:
comment could be adjusted after the rework:
Maybe something like "Join the execution until completion, if at least one evaluation wasn't successful return its error"?